tests/common/dualtor/data_plane_utils.py

```diff
@@ -235,7 +237,7 @@ def t1_to_server_io_test(activehost, tor_vlan_port=None,
 
     yield t1_to_server_io_test
 
-    cleanup(ptfadapter, duthosts_list)
```

Comment:
I think `duthosts_list` was collected so that cleanup can be done on only the host in which the test was executed.
Will cleanup be now done on both duthosts?